### PR TITLE
[dev/arcade-migration] Fix package and shared framework diffs

### PR DESCRIPTION
**Pull request by dagood**
Fixes https://github.com/dotnet/core-setup/issues/7253

### CODE REVIEW

File: src/pkg/packaging-tools/framework.dependency.targets [hunk 1]
```diff
@@ -143,11 +145,11 @@
     </PropertyGroup>
 
     <GenerateFileVersionProps Files="@(SharedFrameworkRuntimeFiles)"
-                              PackageId="$(FrameworkPackageName)"
+                              PackageId="__PackageId__"
```
Comment (dseefeld) [inline src/pkg/packaging-tools/framework.dependency.targets]: Why these items are now hardcoded here?

Comment (dagood) [reply src/pkg/packaging-tools/framework.dependency.targets]: They aren't, this is a templating token that's replaced in the pkgproj. The commit + commit message might shed some light: https://github.com/dotnet/core-setup/pull/7259/commits/cc3c81b6caec5947f03f02ecb306610236e82fee.

Comment (dseefeld) [reply src/pkg/packaging-tools/framework.dependency.targets]: Doh!  If I would have read a little further, I would see where the replacement happens.  Thanks.

Comment (dagood) [reply src/pkg/packaging-tools/framework.dependency.targets]: Would actually be a good place for a comment, on second thought. It's only in context in this PR, not when people read the file itself later.

**Pull request merged**